translate/src/modules/search/components/SearchBox.test.js

```diff
@@ -91,47 +91,6 @@ describe('<SearchBoxBase>', () => {
     expect(wrapper.state().search).toEqual('');
   });
 
-  it('returns correct list of selected statuses', () => {
-    const wrapper = shallow(
-      <SearchBoxBase
-        parameters={{}}
-        project={PROJECT}
-        searchAndFilters={SEARCH_AND_FILTERS}
-      />,
-    );
-
-    wrapper.setState({
-      statuses: {
-        warnings: true,
-        errors: true,
-        missing: false,
-      },
-    });
-
-    const selected = wrapper.instance().getSelectedStatuses();
-    expect(selected).toEqual(['warnings', 'errors']);
-  });
-
-  it('returns correct list of selected extras', () => {
-    const wrapper = shallow(
-      <SearchBoxBase
-        parameters={{}}
-        project={PROJECT}
-        searchAndFilters={SEARCH_AND_FILTERS}
-      />,
-    );
-
-    wrapper.setState({
-      extras: {
-        unchanged: true,
-        rejected: false,
-      },
-    });
-
-    const selected = wrapper.instance().getSelectedExtras();
-    expect(selected).toEqual(['unchanged']);
-  });
-
```

Comment:
I haven't gotten to review this, but if it's easy for you to disclose, why is it good to remove these tests?

Reply:
Because they're testing an internal function that shouldn't be public in the first place, and is inlined in the change. These are the lines of code this is testing, in their entirety: https://github.com/mozilla/pontoon/blob/9faea69c202376f3f3515ef665be04a12b61a130/translate/src/modules/search/components/SearchBox.tsx#L189-L193